### DISCUSSION TITLE
Location vs Events vs Category

### DISCUSSION THREAD

**Discussion by WXQ8NPb5TrbDNqzywcxF** [liedekef/events-made-easy#667] [General]
Hi,

I don't find the way to make it working,

Currently I have a page that display a list of location,  user select a location and after that, a list of events is displayed.

What I want to do is to create 2 categories of events with the sames locations and would like to have two "entry points"
So 1 page will list all location but filter only events to a specific category and a second page that will display all locations but display events only for another category

I tried on page with [eme_locations eventful=true category=4], it show only locations that contain events with this category,  but when I select the location,  all the event of the location is listed,  I dont find the way to filter the event.

Thanks

**Comment by liedekef** [comment]
You need to pass the category as an argument to your event link, adding "?eme_cat_filter=xxx" (with xxx the wanted category) should do it. You will need to use the template_id option of eme_locations for this, so you can change the event link to your liking.
